src/watchful/attributes.py

```diff
@@ -297,7 +297,7 @@ def write(cell_data):
                 # that many of them.
                 span_val.append(attrs[attr])
                 span_val.append(
-                    base64str([values[attr][val] if val else 0 for val in vals])
+                    base64str([values[attr][str(val)] if val else 0 for val in vals])
```

Comment:
This will output `True` and `False`. I'm assuming ingest will just convert it to lowercase, but the capitalization is pretty python specific. I wonder if we could change this to `str(val).lower()` instead.